.github/workflows/nightly-modpack-build.yml

```diff
@@ -156,3 +156,35 @@ jobs:
           path: GTNH-${{ steps.date.outputs.today }}-mmcprism-new-java.zip
           compression-level: 0 # already compressed by Python
 
+      # Pulls the newest changelog filename and stores it in an env variable
+      # so we can use it in the commit
+      - name: Retrieve Changelog Name
+        if: success() || failure()
+        run: |
```

Comment:
Add `shell: bash` to each of the steps here, it ensures `set -eo pipefail` to exit if commands exit with an error code (by default all errors except in the last command are ignored)